### DISCUSSION TITLE
API to Fetch GD Mod Data

### DISCUSSION THREAD

**Discussion by oopshnik** [geode-sdk/sdk#1132] [Geode Ideas]
I'm working on the bot for the GD Mods channel, but it's kind of a mess right now. I was thinking it would be way better if we had an API to fetch all the mod data instead. That way, we could easily get info about mods like descriptions, versions, and download links. It would make the whole thing run a lot smoother.
 
 
Thanks. BR

**Comment by prevter** [comment]
https://api.geode-sdk.org/v1/mods/geode.node-ids
hope this helps

**Reply by dankmeme01** [reply]
you can see the index API docs [here](https://geode-sdk.github.io/server/)

**Reply by dankmeme01** [reply]
wait i didnt mean to reply to prevter but oh well

**Reply by oopshnik** [reply]
> https://api.geode-sdk.org/v1/mods/geode.node-ids hope this helps

thanks

**Reply by oopshnik** [reply]
> wait i didnt mean to reply to prevter but oh well

also thanks
